include/hpp/constraints/iterative-solver.hh

```diff
@@ -148,9 +208,31 @@ namespace hpp {
         /// \name Problem resolution
         /// \{
 
+        /// Solve the system of non linear equations
+        ///
+        /// \param arg initial guess,
+        /// \param ls line search method used.
+        ///
+        /// Use Newton Rhapson like iterative method until the error is below
+        /// the threshold, or until the maximal number of iterations has been
+        /// reached.
+        ///
+        /// \note Explicit constraints are expressed in their implicit
+        ///       form.
```

Comment:
> Explicit constraints are expressed in their implicit form.

What do you mean ?

Reply:
I mean that q2 = f (q1) is replaced by q2 - f (q1) = 0.
Isn't it explicit enough ?